### ISSUE TITLE
loss value is too high, any explanation? thank you.

### ISSUE THREAD

**Issue by MenSanYan** [jfzhang95/pytorch-deeplab-xception#2]
Thank you for your sharing!
Could you please provide the results, such as a screen snapshot.
I find no error in your source code. However, high loss value exists all the time (even after 300 epochs).
What's wrong with that?

Thank you for answering.

**Comment by jfzhang95** [comment]
Printed loss is the sum of loss of each trained image in an epoch without size_average, so I think that's why printed loss is very high.
